frontend/packages/dev-console/src/components/topology/knative-overview.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import {
+  ResourceLink,
+  SidebarSectionHeading,
+  ExternalLink,
+} from '@console/internal/components/utils';
+import { OverviewItem } from '@console/internal/components/overview';
+import { ConfigurationModel, KsRouteModel, RevisionModel } from '../../models';
+
+const OverviewDetailsKnativeResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
```

Comment:
Use `React.FC` and not `React.SFC`.

Create separate files for each component and use default exports.
You can create a separate knative dir outside of topology as the resource details is a separate feature from topology.

Reply:
later once your pr goes in, we can move it under knative-plugin